e2e/tests/test_e2e.py

```diff
@@ -1187,7 +1187,7 @@ def test_major_version_upgrade(self):
         Test major version upgrade: with full upgrade, maintenance window, and annotation
         """
         def check_version():
-            p = k8s.patroni_rest("acid-upgrade-test-0", "")
+            p = k8s.patroni_rest("acid-upgrade-test-0", "") or {}
```

Comment:
or {}?
I don't understand python sometimes

Reply:
it can return None and then `p.get("server_version", 0)` will break everything with exception

`or {}` returns empty dict instead of None